### DISCUSSION TITLE
Has anyone written a python library for the API?

### DISCUSSION THREAD

**Discussion by bjlockie** [transmission/transmission#2461] [Q&A]
Is the API available as a python library?

Has anyone written something to do this:?
I want to load the saved /var/lib/transmission-daemon/info/torrents/*.torrent and files (from transmission-daemon) and save them with the display-name.
Eg. f1fcdc1462d36530f526c1d9402eec9100b7ba18.torrent saved as ubuntu-21.10-desktop-amd64.iso.torrent

**Comment by maxz** [comment]
https://github.com/Trim21/transmission-rpc

**Comment by Petrprogs** [comment]
Visit RPC Specification in docs
